samples/grids/tree-grid/data-summaries-custom/wwwroot/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta name="description" content="Explore samples of IgniteUI for Blazor components.">
+    <meta name="keywords" content="Sample, Example, Blazor, IgniteUI for Blazor, Components, Infragistics">
+    <meta name="author" content="Infragistics">
+    <title>Samples | IgniteUI for Blazor | Infragistics</title>
+    <base href="/" />
+    <link href="https://static.infragistics.com/xplatform/images/browsers/blazor.ico" rel="icon" type="image/x-icon">
+    <link href="https://static.infragistics.com/xplatform/css/samples/shared.v6.css" rel="stylesheet" />
+    <link href="https://static.infragistics.com/xplatform/css/samples/blazor.css" rel="stylesheet" />
+    <link href="_content/IgniteUI.Blazor/themes/grid/light/bootstrap.css" rel="stylesheet" />
+    <link href="index.css" rel="stylesheet" />
+</head>
+
+<body>
+    <app class="app"><img class="app-loading" src="https://static.infragistics.com/xplatform/images/browsers/loading.gif" /></app>
+    <div class="app-error-ui">
+        An unhandled error has occurred.
+        <a href="" class="reload">Reload</a>
+        <a class="app-error-dismiss">ðŸ—™</a>
+    </div>
+    <!-- importing blazor bundle for IG controls: -->
+    <script src="_content/IgniteUI.Blazor/app.bundle.js"></script>
+    <script src="_framework/blazor.webassembly.js"></script>
```

Comment:
You should include the `events.js` file because the custom `WebTreeGridCustomSummary` script will not be executed.
```suggestion
    <script src="_framework/blazor.webassembly.js"></script>
    <script src="events.js"></script>
```